docs/expose-agent.md

```diff
@@ -23,11 +23,17 @@ Determine the public IP of your instance:
 
 Now imagine that your sub-domain is `agent.example.com`, you need to create a DNS A record of `agent.example.com=141.73.80.100`, changing both the sub-domain and IP to your own.
 
-Once created, edit the start.sh file on the agent and add two flags:
+Once the agent is installed, edit /etc/default/actuated on the agent and set the following two variables:
```

Comment:
Can we avoid having the user edit these files?

Why aren't we setting these variables as flags during `install-service`?

Reply:
These can be set with flags during `install-service` and this is actually covered in the agent installation instructions. 

Would it be better to remove the steps here and remove the steps here and just reference the installation steps here?

Reply:
Could we have them instead of editing the files, run the install-service again, would that work?

Now that you've explained it I think we could also leave this section

Reply:
That would work. Only caveat with that would be that if they changed any other parameters, e.g. referenced the kvm kernel image, those flags would also need to be repeated or the variables would be overridden with the defaults.